### PR TITLE
Enabling RTD supported format for docs

### PR DESCRIPTION
**Pull request by kumare3**

### CODE REVIEW

File: rsts/make.bat [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@ECHO OFF
```
Comment (wild-endeavor) [inline rsts/make.bat:1]: can we just delete this file?

File: rsts/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,105 @@
+#
```
Comment (wild-endeavor) [inline rsts/requirements.txt:1]: can you add a make target to generate this file?

File: rsts/conf.py [hunk 3]
```diff
@@ -92,14 +97,32 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
-html_static_path = ['_static']
 html_logo = "flyte_lockup_on_dark.png"
+html_static_path = ['_static']
```
Comment (wild-endeavor) [inline rsts/conf.py:101]:
```suggestion
html_static_path = []
```

Comment (wild-endeavor) [reply rsts/conf.py:101]: if you don't do this you just get a warning every time you build.

File: rsts/user/getting_started/index.rst [hunk 4]
```diff
@@ -14,4 +14,4 @@ Getting Started
 
    examples
    create_first
-   more_examples 
+   Learn Flytekit by example <https://flytecookbook.readthedocs.io/en/latest/>
```
Comment (wild-endeavor) [inline rsts/user/getting_started/index.rst:17]: does this link work?

File: rsts/make.bat (hunk 1 above)
Comment (kumare3) [reply rsts/make.bat:1]: I guess we can, it is auto-generated

File: rsts/requirements.txt (hunk 2 above)
Comment (kumare3) [reply rsts/requirements.txt:1]: I did add it, what happened, - oops overrode it

File: rsts/conf.py (hunk 3 above)
Comment (kumare3) [reply rsts/conf.py:101]: ok

**Review by katrogan — DISMISSED** [review]

File: rsts/user/getting_started/index.rst (hunk 4 above)
Comment (kumare3) [reply rsts/user/getting_started/index.rst:17]: yup

**Review by wild-endeavor — APPROVED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Pull request merged**